src/schemas/execute.yaml

```
@@ -0,0 +1,112 @@
+MulticallArgs:
+  title: Arguments for multi call
+  type: object
+  required:
+    - block
+    - calls
+  properties:
+    block:
+      title: Block tag
+      $ref: '#/components/schemas/BlockNumberOrTag'
+    stateOverrides:
+      title: State overrides
+      $ref: '#/components/schemas/StateOverrides'
+    blockOverrides:
+      title: Overrides for block metadata
+      $ref: '#/components/schemas/BlockOverrides'
+    calls:
+      title: List of calls
+      $ref: '#/components/schemas/Calls'
+Calls:
+  title: List of message calls
+  type: array
+  items:
+    $ref: '#/components/schemas/GenericTransaction'
+StateOverrides:
+  title: Accounts in state to be overridden
+  type: object
+  additionalProperties:
+    $ref: '#/components/schemas/AccountOverride'
+AccountOverride:
+  title: Details of an account to be overridden
+  type: object
+  oneOf:
+    - $ref: '#/components/schemas/AccountOverrideState'
+    - $ref: '#/components/schemas/AccountOverrideStateDiff'
+AccountOverrideState:
+  title: Account override with whole storage replacement
+  properties:
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    state:
+      title: Storage
+      $ref: '#/components/schemas/AccountStorage'
+AccountOverrideStateDiff:
+  title: Account override with partial storage modification
+  properties:
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    stateDiff:
+      title: Storage difference
+      $ref: '#/components/schemas/AccountStorage'
+AccountStorage:
+  title: Storage slots for an account
+  type: object
+  additionalProperties:
+    - $ref: '#/components/schemas/hash32'
+BlockOverrides:
+  title: Context fields related to the block being executed
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    difficulty:
+      title: Difficulty
+      $ref: '#/components/schemas/uint256'
+    time:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    coinbase:
+      title: Coinbase
```

```suggestion
      title: FeeRecipient
```
I believe `fee recipient` is the nomenclature we are using now (as of The Merge) for the person who received the transaction fees in the block.  This is separate from proposer, which I don't *think* the execute layer has access to.